tictactoe.rb

```diff
@@ -0,0 +1,128 @@
+class Game
+    attr_accessor :board, :board_o, :board_x, :b
+
+    def initialize
+        #to intialize -start method and winning positions
+        start
+       
+        @win_positions =
+      [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8], [0,4,8], [2,4,6]]
+    end
+
+    def start
+        #@turns_played = 0
+         @player = "x"
+        @board = ["-", "-", "-", "-", "-", "-", "-", "-", "-"]
```

Comment:
You could achieve an Array holding three Arrays like: `@board = Array.new(3) { Array.new(3) { '-' } }`

Reply:
oh thanks good to know that,  but I need to use it yet